packages/shared/lib/services/sync/run.service.ts

```diff
@@ -383,101 +381,10 @@ export default class SyncRun {
                     await this.finishSync(models, syncStartDate, syncData.version as string, totalRunTime, trackDeletes);
 
                     return { success: true, error: null, response: true };
-                }
+                } else {
+                    const error = new NangoError('sync_script_failure', 'The sync script did not return a void response', 500);
```

Comment:
I'm not sure what your view on this, maybe it's cleaner to create a dedicated error and use a 400? wdyt

Reply:
Will keep it consistent with a 500 for now. We need to take a pass at errors across the entire app so can make decisions to revamp it then